clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -18,6 +18,29 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
     "https://www.googleapis.com/auth/userinfo.email"
   )
 
+  private val shellOutCreds: Either[Throwable, OAuth2Credentials] = {
+    logger.debug("Shelling out to gcloud to get credentials")
+    Right(ExternalGCloudCredentials)
+  }
+
+  def getCredentials(
+    accessToken: Option[AccessToken],
+    serviceAccountJson: Option[Path]
+  ): Either[Throwable, OAuth2Credentials] = {
+    if (accessToken.isEmpty && serviceAccountJson.isEmpty) {
+      Try(GoogleCredentials.getApplicationDefault).recoverWith {
```

Comment:
You can avoid the `Try(...).toEither` dance by importing `cats.syntax.either._`, which gives you `Either.catchNonFatal` (like [here](https://github.com/broadinstitute/clio/blob/d0b2f59e692333a11c7de595c440d381fcb8c741/clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala#L46)).

Reply:
Why aren't these language features...